File: Python-164/chapter.4/lab4.14/lab_program.py
# Instructions are imported from java 164 class
# The syntax && logic flow might not follow exact instructions.
'''
/*
A pedometer treats walking 2,000 steps as walking 1 mile. Write a program whose input is the number of steps, and whose output is the miles walked.

Output each floating-point value with two digits after the decimal point, which can be achieved as follows:
System.out.printf("%.2f", yourValue);

Ex: If the input is:

5345

the output is:

2.67

Your program must define and call a method:
public static double stepsToMiles(int userSteps)
*/
'''

def stepsToMiles(userSteps):
    # print converted steps to miles
    print("{:.2f}".format(float(userSteps) / 2000))

stepsToMiles(input())

File: Java-164/chapter.4/lab4.13/LabProgram.java
/*


One lap around a standard high-school running track is exactly 0.25 miles. Write a program that takes a number of miles as input, and outputs the number of laps.

Output each floating-point value with two digits after the decimal point, which can be achieved as follows:
System.out.printf("%.2f", yourValue);

Ex: If the input is:

1.5

the output is:

6.00

Ex: If the input is:

2.2

the output is:

8.80

Your program must define and call a method:
public static double milesToLaps(double userMiles)
*/
import java.util.Scanner;

public class LabProgram {
   
   public static double milesToLaps(double userMiles) {
      return userMiles / .25;
   }
   public static void main(String[] args) {
      Scanner scnr = new Scanner(System.in);
      double userInput = scnr.nextDouble();
      System.out.printf("%.2f\n",milesToLaps(userInput));
      scnr.close();
   }
}